Marlin/src/lcd/fontutils.cpp

```diff
@@ -143,12 +148,12 @@ uint8_t* get_utf8_value_cb(uint8_t *pstart, read_byte_cb_t cb_read_byte, wchar_t
 
 static inline uint8_t utf8_strlen_cb(const char *pstart, read_byte_cb_t cb_read_byte) {
   uint8_t cnt = 0;
-  uint8_t *pnext = (uint8_t *)pstart;
+  uint8_t *p = (uint8_t *)pstart;
   for (;;) {
-    wchar_t ch;
-    pnext = get_utf8_value_cb(pnext, cb_read_byte, &ch);
```

Comment:
In UTF-8 all "additional" bytes have mask 0b10xxxxxx. No need to decode each char sequence to just count chars.